Markdown/developer_product_guide/structures.md

```diff
@@ -2,77 +2,76 @@
 
-Ansys Rigid Body Dynamics is an analysis system within the Ansys Mechanical Premium and Enterprise products that takes the form of an IronPython library within ACT.
+### Ansys Mechanical and MAPDL training material
 
-### Ansys Mechanical and MAPDL training material 
-
-* [What is APDL?](https://www.ansys.com/blog/what-is-apdl)
-* [Getting Started With PyMAPDL (AIC)](https://innovationspace.ansys.com/product/getting-started-with-ansys-pymapdl/)
-* [Introduction to APDL Scripting (AIC)](https://innovationspace.ansys.com/product/intro-to-ansys-apdl-scripting/)
-* [ML/AI Datasets using PyMAPDL (AIC)](https://innovationspace.ansys.com/product/ml-ai-datasets-using-pymapdl/)
-* [Creating Modeling and Simulation Applications using PyAnsys (AIC)](https://innovationspace.ansys.com/product/developing-webapps-with-pymapdl/)
+* [WAPDL: The Language of Simulation](https://www.ansys.com/blog/what-is-apdl)
+* [Getting Started With Ansys PyMAPDL (AIS)](https://innovationspace.ansys.com/product/getting-started-with-ansys-pymapdl/)
+* [Introduction to Ansys APDL Scripting (AIS)](https://innovationspace.ansys.com/product/intro-to-ansys-apdl-scripting/)
+* [ML/AI Datasets using PyMAPDL (AIS)](https://innovationspace.ansys.com/product/ml-ai-datasets-using-pymapdl/)
+* [Creating Modeling and Simulation Applications using PyAnsys (AIS)](https://innovationspace.ansys.com/product/developing-webapps-with-pymapdl/)
 * [PyMAPDL Tutorial](https://tutorials.mapdl.docs.pyansys.com/tutorials/01-pymapdl.html)
 * [PyAnsys Training: Overview of PyMAPDL and PyMechanical](https://www.youtube.com/watch?v=Qh4Y07OZdms)
-* [PyAnsys Training: PyMADPL Examples and Use Cases](https://www.youtube.com/watch?v=H_i-O712wQE)
+* [PyAnsys Training: PyMAPDL Examples and Use Cases](https://www.youtube.com/watch?v=H_i-O712wQE)
 * [Exploring PyMechanical access methods: A brief overview](https://developer.ansys.com/blog/exploring-pymechanical-access-methods-brief-overview)
 * [Introduction to Ansys ACT in Mechanical (ALH)](https://www.ansys.com/training-center/course-catalog/structures/introduction-to-ansys-act-mechanical)
 * [Ansys Mechanical Scripting (ALH)](https://learninghub.ansys.com/learn/courses/902/ansys-mechanical-scripting)
-* [Introduction to ACT Wizards (ALH)](https://www.ansys.com/training-center/course-catalog/structures/introduction-to-ansys-act-wizards)
-* [Create a report from a saved Mechanical session](https://developer.ansys.com/blog/create-report-saved-mechanical-session-mechdb-or-mechdat)
+* [Introduction to Ansys ACT Wizards (ALH)](https://www.ansys.com/training-center/course-catalog/structures/introduction-to-ansys-act-wizards)
+* [Create a report from a saved Mechanical session (.mechdb or .mechdat)](https://developer.ansys.com/blog/create-report-saved-mechanical-session-mechdb-or-mechdat)
 
-### Non-Ansys Mechanical and MAPDL training material 
+### Non-Ansys Mechanical and MAPDL training material
 
 * [PADT Blog (PADT)](https://www.padtinc.com/blog-home/)
```

Comment:
@jgd10 What purpose does this link to the PADT home page provide? I think it should be deleted.